<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="selectExecutionPath">
       <property name="text">
        <string>Select Exectuion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ExecutionPath"/>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="runListBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>231</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="elementListBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>231</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="versionComboBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
